Sources/Attribution/AttributionFetcher.swift

```diff
@@ -85,6 +85,13 @@ class AttributionFetcher {
 #if canImport(AdServices)
         do {
             #if targetEnvironment(simulator)
+            #if DEBUG
+            if let mockToken = ProcessInfo.mockAdServicesToken {
+                Logger.warn(Strings.attribution.adservices_mocking_token(mockToken))
```

Comment:
Added this so it becomes obvious if we mess it up and mock it when we shouldn't.